lib/babe/babe_integration_test.go

```diff
@@ -50,10 +51,17 @@ var (
 	}
 )
 
-func createTestService(t *testing.T, cfg ServiceConfig) *Service {
+func createTestService(t *testing.T, cfg ServiceConfig, useWestend bool) *Service {
```

Comment:
could we instead inject the gen+genTrie+genHeader to this helper function? So we move one inch closer to #2336 and also it would be easier to change when adding more runtime genesis

Reply:
Yeah i like this. Let me finish fixing these babe tests, then depending on size of PR will either do here or in different PR

Reply:
Check out Eclesio's requested changes first, we might not need those changes at all

Reply:
Even with eclesio's suggestion I decided to incase we want to test with other runtimes in the future